src/Microsoft.AspNetCore.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -78,9 +137,13 @@ protected virtual bool CanUpdateProperty(ModelMetadata propertyMetadata)
             return CanUpdatePropertyInternal(propertyMetadata);
         }
 
-        internal bool CanCreateModel(MutableObjectBinderContext context)
+        protected virtual Task BindProperty(ModelBindingContext bindingContext)
```

Comment:
Will add docs. You can override this to do something special for a specific property on your model.